Formula/dar.rb

```diff
@@ -27,11 +40,18 @@ class Dar < Formula
   depends_on "upx" => [:build, :optional]
 
   def install
+    if build.stable? && ENV.compiler == :clang && MacOS.version >= :mavericks
+      ENV.libstdcxx
+    elsif build.devel?
+      ENV.O0
+      ENV.deparallelize
```

Comment:
I just closed the VM down to work on something else 🙈. Unless something changed in today/yesterday's (depending on timezone) Xcode, I know `gcc` builds fine on Sierra at least.